### ISSUE TITLE
pass working fine but qtprocess failure with qtpass

### ISSUE THREAD

**Issue by Bulmer** [IJHack/QtPass#260]
I really want to use qtpass, because of its extra options, but whenever i start qtpass it asks me to create a new key pair, hangs on generation and ultimately fails qprocess. I already have my primary keys setup via gnupg and a useable .password-store via pass.

**Comment by annejan** [comment]
Can you tell me more about your setup, are you running macOS or a Linux distribution (if so which).
Or are you running pass on Windows with cygwin or the new bash layer?

Also which version of QtPass are you trying to run.

My first guess is a mismatch between the gpg version pass uses and QtPass uses.
Possibly some other environmental issue.

**Comment by Bulmer** [comment]
Hello annejan. 
I am using Sparkylinux which is a rolling distribution based on Debian Testing. Versions are:
gnupg 2.1.16
pass 1.6.5
qtpass 1.1.6

# I just realised that if I start qtpass from a root terminal it goes straight to the Configuration box, whereas if I start it from my user terminal it wants me to create a new gpg pair and the Config box says qprocess fail. I have created my primary keys under my User name.

**Comment by annejan** [comment]
Could still be a `gpg` and `gpg2` missmatch . .

https://github.com/IJHack/QtPass/blob/master/FAQ.md#i-have-an-other-issue-with-gpg

Hope that helps, I might give Sparkylinux a try in a VM this evening . .

**Comment by Bulmer** [comment]
gpg --list-keys and gpg2 --list-keys list the same keys (also --list-secret-keys)

**Comment by Bulmer** [comment]
Not sure if this was a bug as it seems to be an ownership issue, but I now have it working perfectly under my user account now. As root I edited /home/bulmer/.config/LJHack/QtPass.conf . I Changed "passStore=/root/.password-store/" to "passStore=/home/bulmer/.password-store/".

**Comment by berndhahnebach** [comment]
This helped a lot :-)

I have a debian system which I upgraded a from ealier debian version eventually to buster. Than I completely reinstalled buster and used my old home folder. Pass worked whereas QtPass refused to work. I had to edit the QtPass.conf in a way to set gpg executable from gpg2 to gpg. I might could have made a link from gpg2 to gpg too but editing is more future proof IMHO.

cheers bernd
